packages/blockly-field-date/test/test.js

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
Prefer file name index.js over test.js. I'm mostly having an issue with ending with "test", because that's typically the naming convention for unit/integration test files.

Reply:
Sure. index.js sounds good.